api/src/models/ImageFile.ts

```diff
@@ -7,6 +8,7 @@ class ImageFile {
         "MEDIUM": 640,
         "THUMBNAIL": 120
     };
+    sizeArray: Array<string> = ["LARGE","MEDIUM","THUMBNAIL"];
```

Comment:
Rather than duplicating information here, can you just pull the keys from sizes instead of using a separate array?

Reply:
Updated!